app/javascript/src/stacked-menu.js

```diff
@@ -0,0 +1,927 @@
+/**
+ * A flexible stacked navigation menu.
```

Comment:
I think this file is already in here if you have rebased recently

Reply:
Delete?